### ISSUE TITLE
Image placement changes - open vs download

### ISSUE THREAD

**Issue by atdiff** [bpampuch/pdfmake#557]
My document has a very basic docDefinition of 

> var url = canvas.toDataURL('image/JPEG',1);
> var docDefinition = {
>  pageOrientation: 'landscape',
>   content: [
>    {text: 'Header'},
>     {
>       image: url
>     },
>    {text: 'Footer'}
>   ]
> }; 
> 
> var doc = pdfMake.createPdf(docDefinition);
> doc.open();
> doc.download();

When I call doc.open(), the image is left-aligned with the text.  When I use doc.download(), the image is shifted to the right about to the end of the text (10px or so).

Is there any reason why the positioning would change?

**Comment by stodolos** [comment]
I have a very similar issue...

I have an image inside of a column (b/c I want it centered along with other text on left and right), and the image is always **aligned all the way to the right using both download and open**.  **On print, the image doesn't even show up.**

Another thing to note is that I had to put the image in a stack inside the column.  **Putting the image inside of the column directly did not work.**

**Comment by stodolos** [comment]
I just tested out a bunch of different ways to try and get the image to stay where I wanted it to, and after A LOT spinning my wheels there, I remembered that taking an subsequent action (i.e. download, open, print) caused a different result.  I was storing my result in the $scope, and then just calling the action I wanted, but that was the issue...

If I re-generate the PDF for each action I call, then it works fine!
